### PR TITLE
Add support for PHPUnit 8

### PR DESCRIPTION
**Pull request by emodric**
* Adds support for PHPUnit 8
* Drops support for PHP 7.1
* Drops support for PHPUnit 7

Requires https://github.com/SymfonyTest/SymfonyConfigTest/pull/59 for tests to pass

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="./vendor/autoload.php" colors="true" strict="true">
+<phpunit bootstrap="./vendor/autoload.php" colors="true">
```
Comment (greg0ire) [inline phpunit.xml.dist]: An improvement would be to apply these changes: https://github.com/SymfonyTest/SymfonyDependencyInjectionTest/pull/110/files#diff-98c12a693941ae3c4f50872403416471

Comment (emodric) [reply phpunit.xml.dist]: Nice. Will fix first thing Monday!

Comment (greg0ire) [reply phpunit.xml.dist]: I can make a PR on your PR so you just have to click merge !

Comment (greg0ire) [reply phpunit.xml.dist]: Here you go, please merge : https://github.com/emodric/SymfonyDependencyInjectionTest/pull/1

Comment (emodric) [reply phpunit.xml.dist]: Done!

**Comment by enumag** [comment]
Any news on this?

**Comment by greg0ire** [comment]
> the config counterpart is merged.

This just happened, please update the `composer.json` accordingly!

**Comment by emodric** [comment]
@greg0ire Done!

**Comment by emodric** [comment]
Will fix pending errors tomorrow.

**Comment by emodric** [comment]
The build is green :+1:

**Comment by greg0ire** [comment]
@mwadon @Nyholm can you please merge and release this?

**Comment by xabbuh** [comment]
That would be 4.0 then, right? Should we add a branch alias to the `composer.json`?

**Comment by Nyholm** [comment]
Yes. Correct

**Comment by emodric** [comment]
@nyholm Do you want me to add it? Or you will before the release?

**Comment by greg0ire** [comment]
@emodric IMO you should add it. Don't let people release this as 3.x by mistake.

**Comment by emodric** [comment]
OK, done!

**Pull request merged**